File: src-gen/BookingSystemV4/BookingSystemV4/ClientApp/src/api/models/bookings/MovieTicket.ts
import {RegularSeatSchedule} from "../schedules/RegularSeatSchedule"
import {Seat} from "../resources/Seat"
import {Cust1} from "../customers/Cust1"
export type MovieTicket = {
	id: string
	seatSchedule: RegularSeatSchedule
	seat: Seat
	cust: Cust1
} 

File: src-gen/BookingSystemV4/BookingSystemV4/Persistence/Models/MovieTicket.cs
using System;
using System.Collections.Generic;

namespace BookingSystemV4.Persistence.Models
{
    public class MovieTicket : IEntity
    {
    	public Guid Id {get; set;}
        public RegularSeatSchedule seatSchedule {get; set;} 
        public Seat seat {get; set;} 
        public Cust1 cust {get; set;} 
    }
}
